backends/arm/test/models/test_mobilenet_v2_arm.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
MV2 test (though skipped) but nice to see :)

Reply:
Yeah, hopefully we'll enable them one-by-one soon :)